src/main/groovy/com/blackducksoftware/integration/hub/detect/hub/HubManager.groovy

```diff
@@ -173,6 +172,10 @@ class HubManager implements ExitCodeReporter {
         builder.setProjectLevelAdjustments(detectConfiguration.getProjectLevelMatchAdjustments())
         builder.setPhase(detectConfiguration.getProjectVersionPhase())
         builder.setDistribution(detectConfiguration.getProjectVersionDistribution())
+        int projectTier = detectConfiguration.projectTier
+        if (projectTier > 0 && projectTier <= 5) {
+            builder.setProjectTier(projectTier)
```

Comment:
Please remove the if check and only leave `builder.setProjectTier(detectConfiguration.projectTier)`